core/parcel-transformer-manifest/src/schema.ts

```diff
@@ -185,7 +185,11 @@ const commonProps = {
           type: "string",
           enum: ["document_idle", "document_start", "document_end"]
         },
-        all_frames: booleanSchema
+        all_frames: booleanSchema,
+        world: {
+          type: "string",
+          enum: ["MAIN", "ISOLATED"]
+        }
```

Comment:
@flexchar - might need to double check if this is available on Firefox yet. If not, let's add this, but also keep the old behavior for cross-browser compatibility I think :-?

Reply:
Actually would be best to do the drop console in a separate PR to merge quickly, then we can do the main world fix later :-?

Reply:
I will do that!